src/jailer/src/main.rs

```diff
@@ -140,6 +141,12 @@ impl fmt::Display for Error {
             CStringParsing(_) => write!(f, "Encountered interior \\0 while parsing a string"),
             Dup2(ref err) => write!(f, "Failed to duplicate fd: {}", err),
             Exec(ref err) => write!(f, "Failed to exec into Firecracker: {}", err),
+            ExecFileName(ref filename) => write!(
+                f,
+                "Invalid filename. The filename of `--exec-file` option must contain \
+                 \"firecracker\": {}",
+                filename
+            ),
             FileName(ref path) => write!(
```

Comment:
We should also add an test for the new error format inside [test_error_display](https://github.com/firecracker-microvm/firecracker/blob/main/src/jailer/src/main.rs#L486) unit test.